pkg/common/common.go

```diff
@@ -255,3 +259,38 @@ func DatabaseLevelResourceGroups(kvs []*commonpb.KeyValuePair) ([]string, error)
 
 	return nil, fmt.Errorf("database property not found: %s", DatabaseResourceGroups)
 }
+
+func CollectionLevelReplicaNumber(kvs []*commonpb.KeyValuePair) (int64, error) {
```

Comment:
just use `funcutil.GetAttrByKeyFromRepeatedKV`

Reply:
We can't just use funcutil.GetAttrByKeyFromRepeatedKV to replace CollectionLevelReplicaNumber or CollectionLevelResourceGroups because we aim to unify the parsing logic for specified keys.